stacked-predictions/script.whizzml

```diff
@@ -19,23 +19,10 @@
                                           "prediction_name" name})))
 
 ;; Auxiliary function to create the set of stack models
-(define (create-stack-models train-id obj-id)
-  (if (= ((fetch train-id) ["fields" obj-id "optype"]) "numeric")
-    [(create-model {"dataset" train-id})
-     (create-ensemble {"dataset" train-id
-                       "randomize" false})
-     (create-ensemble {"dataset" train-id
-                       "randomize" true})
-     (create-ensemble {"dataset" train-id
-                       "boosting" {"iterations" 10}})]
-    [(create-model {"dataset" train-id})
-     (create-ensemble {"dataset" train-id
-                       "randomize" false})
-     (create-ensemble {"dataset" train-id
-                       "randomize" true})
-     (create-ensemble {"dataset" train-id
-                       "boosting" {"iterations" 10}})
-     (create-logisticregression {"dataset" train-id})]))
+(define (create-stack-models train-id model-name param)
+  (cond (= model-name "model") (create-model train-id param)
+        (= model-name "ensemble") (create-ensemble train-id param)
+        (= model-name "lr") (create-logisticregression train-id param)))
```

Comment:
i would use here the "official name" of the resource, "logisticregression", maybe also accepting "lr" as an alias?

i'd also add a default clause at the end with `(raise "Predictor name not recognized: " model-name)`